sormas-rest/src/main/java/de/symeda/sormas/rest/CaseResource.java

```diff
@@ -88,6 +89,12 @@
 		return savePushedDto(dtos, FacadeProvider.getCaseFacade()::save);
 	}
 
+	@POST
+	@Path("/post")
```

Comment:
Please rename the endpoint to `pushWithPerson` (to be in line with the existing POST endpoint)